libs/ktor2client-onbehalfof/src/main/kotlin/DownstreamResourceClass.kt

```diff
@@ -145,7 +145,7 @@ class DownstreamResourceClient(
                     Ok(result.body())
                 },
                 onFailure = { error ->
-                    logger.error("received error from downstream api: delete", error)
+                    logger.debug("received error from downstream api: delete", error)
```

Comment:
Vet ikke om det er helt hensiktsmessig å bruke debug, siden root-loglevel settes som "INFO" i alle appene i dette repoet i alle fall. Da ville jeg heller ha fjernet det eller bumpet til debug

Reply:
+1

Reply:
Eg trudde nettopp det var poenget, at vi skrur det ned her for å kunne skru det opp ved behov, men ikkje bruke det til vanleg. Viss ikkje, så er eg einig.

Reply:
Var det som var poenget - at vi kan sette til debug ved behov - men slippe å spamme loggene til vanlig. Men med trafikken vår kan det hende at det er like greit å sette på tilsvarende logging som mellom tjenester (altså kall med returnert status). Det kan gi oss god kontroll nå i starten.